packages/nextjs/src/client/routing/appRouterRoutingInstrumentation.ts

```diff
@@ -64,11 +59,8 @@ export function appRouterInstrumentation(
 
       startNavigationSpanCallback({
         name: newPathname,
-        tags: {
-          ...DEFAULT_TAGS,
-          from: currPathname,
-        },
         attributes: {
+          from: currPathname,
```

Comment:
@lforst Just a reminder for ourselves during the v8 alpha: we need to review all our auto instrumentation for what attribute we set, what we actually need/want, etc.